### ISSUE TITLE
[BUG] Unity Web Player Games Don't work on Steam Deck

### ISSUE THREAD

**Issue by TheDoomCider** [oblivioncth/CLIFp#59]
**Can't launch Unity Web Player games on Steam Deck through CLIFp. I can launch html and flash games without issue. I am having the same issue on both Steam OS (using proton) and Windows 10. I also tried using both the normal flashpoint launcher and LaunchBox on Windows 10 and Unity Web Player games work. I am having no issues on my Windows 10 Intel gaming PC so the issue seems to be Steam Deck specific.**

**To Reproduce**
1. Downloaded CLIFP and Flashpoint Infinity 11.1
2. Moved the CLIFP.exe to the Flashpoint Infinity folder
3. Commands used: play -i 2abf6233-9c95-4e81-8196-722b17faa8eb
run --app="FPSoftware\startUnity.bat" --param="2.x http://y8.com/ben_10_ultimate_crisis/ben_10_ultimate_crisis.html"
4. The game downloads and the error occurs during the mounting process.

**Expected behavior**
Game launching without issue

**Screenshots**
![unity web error](https://user-images.githubusercontent.com/24508166/235660167-0f42cfad-a430-4818-9766-4bf7c7b1fb53.PNG)

Version:
 - CLIFp version: 0.9.4_Windows_Static_x64
- Flashpoint version: Infinity 11.1 (Windows version)

**Comment by oblivioncth** [comment]
This might be because of #57, let me know if this specific build works for you: https://github.com/oblivioncth/CLIFp/suites/12579933733/artifacts/672694390

**Comment by TheDoomCider** [comment]
Ok, the Steam OS error seems to be an issue with Proton. I got the same issue when I tried using Ubuntu on my Gaming PC.

**Comment by oblivioncth** [comment]
Moving forward use this build to try anything until the next release, that last one I linked ended up having a different issue: https://github.com/oblivioncth/CLIFp/suites/12655626402/artifacts/678241068

For the Windows attempt, the log indicates a permissions denied error, specifically from FlashpointSecurePlayer.exe, which is ran by the batch file associated with Unity Web Player Games. Not sure what potential differences you have between your OS configuration on your Steam Deck and your desktop, but make sure that the entire flashpoint directory (including sub-folders) are set to "Full Control" for the Security Properties of the account you're running CLIFp/Flashpoint under.

**Comment by TheDoomCider** [comment]
Interesting! Thanks!

**Comment by oblivioncth** [comment]
@TheDoomCider We you able to get the games working on your Steam Deck (on Windows of course)?

**Comment by TheDoomCider** [comment]
Yes, all I did was copy the game commands that were in launchbox and paste them it into playnite (which is the launcher I'm using for my windows games). The games are launching without issue now.

**Comment by oblivioncth** [comment]
Hmm, strange that LB had an issue with it. But since you aren't actually using that, I won't bother trying to find the needle in the haystack in this instance.

Glad you're back on track.
